### ISSUE TITLE
Error running go

### ISSUE THREAD

**Issue by paulonteri** [paulonteri/remote-code-execution-environment#12]
**Describe the bug**
Error while running go in a Docker container

**Example Error**
Command failed: cd ../../35b4cdd0-c872-11ea-b700-bb767543deb2 && go run main.go
![go_error](https://user-images.githubusercontent.com/45426293/87831387-0ac6d180-c88c-11ea-812e-99398f2c0171.png)
 
@ItsCosmas you can try fixing this from #8

**Comment by ItsCosmas** [comment]
Hey, how did you reproduce this issue? I can't seen to reproduce it on my side.

**Comment by ItsCosmas** [comment]
> Hey, how did you reproduce this issue? I can't seen to reproduce it on my side.

Go seems to be working in production as of now.

**Comment by paulonteri** [comment]
> Hey, how did you reproduce this issue? I can't seen to reproduce it on my side.

I just ran the boilerplate code.

**Comment by paulonteri** [comment]
> > Hey, how did you reproduce this issue? I can't seen to reproduce it on my side.
> 
> Go seems to be working in production as of now.

I was playing around with it and also noticed that. Maybe it's a server issue(cache, e.t.c). or this https://github.com/paulonteri/remote-code-execution-environment/pull/10#issuecomment-660343552

The programming gods must not be on our side. 😂😂

Do you mind making another PR with the recent code @ItsCosmas?

**Comment by ItsCosmas** [comment]
I have found the issue. Installing go on Alpine based docker image raises the error `` cannot find runtime/cgo`` ,( see https://stackoverflow.com/questions/55932233/cannot-find-runtime-cgo-in-alpine) everytime you try to run or build go binaries. With run the error doesn't stop execution but is still raised. However using the `go:alpine` image as base instead seems to fix the issue and everything else works well. Node, Python and Java.

Here's a Dockerfile that's fixing this issue:

```
FROM golang:alpine

RUN apk add --no-cache nodejs yarn
RUN apk add --no-cache openjdk8
ENV JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk
ENV PATH="$JAVA_HOME/bin:${PATH}"
RUN apk add --no-cache python3
RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing mono

WORKDIR /usr/src/app

COPY ./core/package.json ./
COPY ./core/yarn.lock ./

RUN yarn --production

COPY ./core .

ENV NODE_ENV="production"
ENV PORT=8080

EXPOSE 8080

CMD [ "node", "Server.js" ]
```

A better suggestion would be use a different base image. Perhaps ubuntu and install all the dependencies there.

**Comment by ItsCosmas** [comment]
> I have found the issue. Installing go on Alpine based docker image raises the error ` cannot find runtime/cgo` ,( see https://stackoverflow.com/questions/55932233/cannot-find-runtime-cgo-in-alpine) everytime you try to run or build go binaries. With run the error doesn't stop execution but is still raised. However using the `go:alpine` image as base instead seems to fix the issue and everything else works well. Node, Python and Java.
> 
> Here's a Dockerfile that's fixing this issue:
> 
> ```
> FROM golang:alpine
> 
> RUN apk add --no-cache nodejs yarn
> RUN apk add --no-cache openjdk8
> ENV JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk
> ENV PATH="$JAVA_HOME/bin:${PATH}"
> RUN apk add --no-cache python3
> RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing mono
> 
> WORKDIR /usr/src/app
> 
> COPY ./core/package.json ./
> COPY ./core/yarn.lock ./
> 
> RUN yarn --production
> 
> COPY ./core .
> 
> ENV NODE_ENV="production"
> ENV PORT=8080
> 
> EXPOSE 8080
> 
> CMD [ "node", "Server.js" ]
> ```
> 
> A better suggestion would be use a different base image. Perhaps ubuntu and install all the dependencies there.

Let me make a new pull request.

**Comment by ItsCosmas** [comment]
Pull Request made fixing golang issue inside docker file. https://github.com/paulonteri/remote-code-execution-environment/pull/13

**Comment by paulonteri** [comment]
Had similar issues with java

**Comment by ItsCosmas** [comment]
> Had similar issues with java

I tested the new image with all the supported languages and they run well without any warnings or errors.
https://github.com/paulonteri/remote-code-execution-environment/pull/13

**Comment by paulonteri** [comment]
Fixed by #13
